Pixel Game/Assets/Scripts/Enemy scripts/EnemyHealth.cs

```diff
@@ -22,15 +24,14 @@ protected override void Start()
     {
         base.Start();
         spriteRenderer = GetComponent<SpriteRenderer>();
-        enemyAttack = gameObject.GetComponent<EnemyAttack>();
-        enemyMovement = gameObject.GetComponent<EnemyMovement>();
+        enemyAttack = gameObject.GetComponent<AiAttack>();
+        enemyMovement = gameObject.GetComponent<AiMovement>();
         colliderList = gameObject.GetComponentsInChildren<Collider2D>();
```

Comment:
Why do we have a list of colliders?

Reply:
Because we should disable all colliders when the enemy is killed and the death animation is playing.